examples/visualisation/visualise_data.ipynb

```diff
@@ -60,7 +62,7 @@
    "outputs": [],
    "source": [
     "# set env variable for data\n",
-    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/tmp/l5kit_data\"\n",
+    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/Users/yourName/Documents/LyftData/\"\n",
```

Comment:
mmmm....if we want to suggest the user to put here his/her path I suggest something like:
`PATH_TO_YOUR_DATA`